feat: add account / staking tool

Sanghren opened the pull request:
Lot of stuff in here, mainly adding a new screen that will for a given user : 
- Display linked addresses on the x-chain 
- Display linked accounts on the p-chain
- Allow export and import (both x and p chain)
- Allow the creation of new x-chain address and p-chain account 
- Easily export from a XChain address to an PChain account .
- Easily export from a PChain account to a XChain address .
- Replace Slopes by Avalanche

Sanghren committed:
feat: Add `Account.vue` with a lot of stuff in it (WIP)

Sanghren committed:
feat: Add `Account.vue` with a lot of stuff in it (WIP)

Sanghren committed:
Merge remote-tracking branch 'origin/master' into feat/account

# Conflicts:
#	package.json
#	yarn.lock

Sanghren committed:
Merge branch 'master' into feat/account

Sanghren committed:
feat: Lot of stuff

Sanghren committed:
Merge branch 'master' into feat/account

Sanghren committed:
feat: Improve error handling

Sanghren committed:
feat: Improve design a bit

Sanghren committed:
feat: Improve design a bit

Sanghren committed:
Merge branch 'master' into feat/account

Sanghren committed:
feat: Move to Avalanche (New name of Slopes)

Sanghren committed:
feat: Some more improvements

tbrunain commented:
Do not merge until this https://github.com/ava-labs/avalanche.js/issues/45 is solved ! (Will maybe need adaptation in code here)

Sanghren committed:
feat: Reenable linter

Sanghren committed:
chore: Some cleanup

Sanghren committed:
feat: Some more cleanup / fixes

Sanghren committed:
Merge branch 'master' into feat/account

Sanghren committed:
Merge branch 'master' into feat/account

# Conflicts:
#	package.json

Sanghren committed:
Merge branch 'master' into feat/account

Sanghren committed:
feat: Add Subnets view

Sanghren committed:
feat: Add jdenticon

Sanghren committed:
Merge branch 'master' into feat/account

# Conflicts:
#	src/AVA.ts

Sanghren committed:
feat: Display validating info

Sanghren committed:
feat: Display validating info

dependabot-preview[bot] committed:
chore(deps-dev): bump vue-cli-plugin-vuetify from 2.0.5 to 2.0.6

Bumps [vue-cli-plugin-vuetify](https://github.com/vuetifyjs/vue-cli-plugin-vuetify/tree/HEAD/packages/vue-cli-plugin-vuetify) from 2.0.5 to 2.0.6.
- [Release notes](https://github.com/vuetifyjs/vue-cli-plugin-vuetify/releases)
- [Changelog](https://github.com/vuetifyjs/vue-cli-plugins/blob/master/packages/vue-cli-plugin-vuetify/CHANGELOG.md)
- [Commits](https://github.com/vuetifyjs/vue-cli-plugin-vuetify/commits/vue-cli-plugin-vuetify@2.0.6/packages/vue-cli-plugin-vuetify)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
chore(deps): bump vuex-smart-module from 0.4.0 to 0.4.2

Bumps [vuex-smart-module](https://github.com/ktsn/vuex-smart-module) from 0.4.0 to 0.4.2.
- [Release notes](https://github.com/ktsn/vuex-smart-module/releases)
- [Commits](https://github.com/ktsn/vuex-smart-module/compare/v0.4.0...v0.4.2)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
chore(deps-dev): bump sass from 1.26.8 to 1.26.9

Bumps [sass](https://github.com/sass/dart-sass) from 1.26.8 to 1.26.9.
- [Release notes](https://github.com/sass/dart-sass/releases)
- [Changelog](https://github.com/sass/dart-sass/blob/master/CHANGELOG.md)
- [Commits](https://github.com/sass/dart-sass/compare/1.26.8...1.26.9)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
chore(deps-dev): bump @typescript-eslint/eslint-plugin

Bumps [@typescript-eslint/eslint-plugin](https://github.com/typescript-eslint/typescript-eslint/tree/HEAD/packages/eslint-plugin) from 3.4.0 to 3.5.0.
- [Release notes](https://github.com/typescript-eslint/typescript-eslint/releases)
- [Changelog](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/CHANGELOG.md)
- [Commits](https://github.com/typescript-eslint/typescript-eslint/commits/v3.5.0/packages/eslint-plugin)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
chore(deps): bump vuex from 3.4.0 to 3.5.1

Bumps [vuex](https://github.com/vuejs/vuex) from 3.4.0 to 3.5.1.
- [Release notes](https://github.com/vuejs/vuex/releases)
- [Changelog](https://github.com/vuejs/vuex/blob/dev/CHANGELOG.md)
- [Commits](https://github.com/vuejs/vuex/compare/v3.4.0...v3.5.1)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
chore(deps-dev): bump @typescript-eslint/parser from 3.4.0 to 3.5.0

Bumps [@typescript-eslint/parser](https://github.com/typescript-eslint/typescript-eslint/tree/HEAD/packages/parser) from 3.4.0 to 3.5.0.
- [Release notes](https://github.com/typescript-eslint/typescript-eslint/releases)
- [Changelog](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/parser/CHANGELOG.md)
- [Commits](https://github.com/typescript-eslint/typescript-eslint/commits/v3.5.0/packages/parser)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
chore(deps-dev): bump eslint-plugin-vue

Bumps [eslint-plugin-vue](https://github.com/vuejs/eslint-plugin-vue) from 7.0.0-alpha.6 to 7.0.0-alpha.8.
- [Release notes](https://github.com/vuejs/eslint-plugin-vue/releases)
- [Commits](https://github.com/vuejs/eslint-plugin-vue/compare/v7.0.0-alpha.6...v7.0.0-alpha.8)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
chore(deps): bump vuetify from 2.3.1 to 2.3.3

Bumps [vuetify](https://github.com/vuetifyjs/vuetify/tree/HEAD/packages/vuetify) from 2.3.1 to 2.3.3.
- [Release notes](https://github.com/vuetifyjs/vuetify/releases)
- [Commits](https://github.com/vuetifyjs/vuetify/commits/v2.3.3/packages/vuetify)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
chore(deps): bump vee-validate from 3.3.5 to 3.3.7

Bumps [vee-validate](https://github.com/logaretm/vee-validate) from 3.3.5 to 3.3.7.
- [Release notes](https://github.com/logaretm/vee-validate/releases)
- [Commits](https://github.com/logaretm/vee-validate/compare/v3.3.5...v3.3.7)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
chore(deps-dev): bump typescript from 3.9.5 to 3.9.6

Bumps [typescript](https://github.com/Microsoft/TypeScript) from 3.9.5 to 3.9.6.
- [Release notes](https://github.com/Microsoft/TypeScript/releases)
- [Commits](https://github.com/Microsoft/TypeScript/commits)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

Sanghren committed:
feat: Too many things

Sanghren committed:
feat: Improve metrics displayed when bootstrapping is in progress on dashboard view

Sanghren committed:
Merge remote-tracking branch 'origin/dependabot/npm_and_yarn/eslint-plugin-vue-7.0.0-alpha.8' into feat/account

Sanghren committed:
Merge remote-tracking branch 'origin/dependabot/npm_and_yarn/sass-1.26.9' into feat/account

# Conflicts:
#	package.json

Sanghren committed:
Merge remote-tracking branch 'origin/dependabot/npm_and_yarn/typescript-3.9.6' into feat/account

Sanghren committed:
Merge remote-tracking branch 'origin/dependabot/npm_and_yarn/typescript-eslint/eslint-plugin-3.5.0' into feat/account

Sanghren committed:
Merge remote-tracking branch 'origin/dependabot/npm_and_yarn/typescript-eslint/parser-3.5.0' into feat/account

# Conflicts:
#	package.json
#	yarn.lock

Sanghren committed:
Merge remote-tracking branch 'origin/dependabot/npm_and_yarn/vee-validate-3.3.7' into feat/account

# Conflicts:
#	package.json

Sanghren committed:
Merge remote-tracking branch 'origin/dependabot/npm_and_yarn/vue-cli-plugin-vuetify-2.0.6' into feat/account

# Conflicts:
#	package.json

Sanghren committed:
Merge remote-tracking branch 'origin/dependabot/npm_and_yarn/vuetify-2.3.3' into feat/account

# Conflicts:
#	package.json
#	yarn.lock

Sanghren committed:
Merge remote-tracking branch 'origin/dependabot/npm_and_yarn/vuex-3.5.1' into feat/account

# Conflicts:
#	package.json

Sanghren committed:
Merge remote-tracking branch 'origin/dependabot/npm_and_yarn/vuex-smart-module-0.4.2' into feat/account

# Conflicts:
#	package.json
